streaming/src/main/scala/org/apache/spark/streaming/util/WriteAheadLogUtils.scala

```diff
@@ -39,6 +39,7 @@ private[streaming] object WriteAheadLogUtils extends Logging {
 
   val DEFAULT_ROLLING_INTERVAL_SECS = 60
   val DEFAULT_MAX_FAILURES = 3
+  val WAL_CLOSE_AFTER_WRITE = "spark.streaming.writeAheadLog.closeAfterWrite"
```

Comment:
yeah yeah ... i shortened to make the point .. .sorry if that wasnt clear. 
@brkyvz receivers by default do not use WAL